compiler/fir/analysis-tests/testData/extendedCheckers/RedundantModalityModifierChecker.kt

```diff
@@ -1,3 +1,27 @@
+// Interface
+interface Interface {
+    // Redundant
+    <!REDUNDANT_MODALITY_MODIFIER!>open<!> val gav: Int
+        get() = 42
+    // Redundant
+    <!REDUNDANT_MODALITY_MODIFIER!>abstract<!> fun foo()
+    // Redundant
+    private <!REDUNDANT_MODALITY_MODIFIER!>final<!> fun bar() {}
```

Comment:
Please don't report `REDUNDANT_MODALITY_MODIFIER` for `final` in `interface`, because this is an error.